src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int SIDE_LIMIT = 1;
+    private static final int SIDE_DEFAULT = 10;
+    private static final int FIGURE_TOTAL = 4;
+    private static final String COLOR_DEFAULT = Colors.WHITE.name();
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        int indexFigure = random.nextInt(FIGURE_TOTAL);
+        switch (indexFigure) {
+            case (0):
+                return new Square(getRandomColor(), getRandomInt(SIDE_LIMIT));
```

Comment:
```suggestion
                return getRandomSquare();
```
lets make private methods to get each figure, this will make your ode much easier to read